### ISSUE TITLE
High Internal Storage Usage

### ISSUE THREAD

**Issue by randoogle** [syncthing/syncthing-android#743]
In the last month or so, I've noticed that SyncThing has started using a lot of my phone's internal storage (almost 300MB). None of my synced folders are on internal storage, they all go to my SD card. I'm assuming that this is some new logging feature. If so, is there a way to limit its usage?
### Version Information

    App Version: 0.8.3
    Syncthing Version: ?
    Android Version: Android 5 (lollipop)

**Comment by licaon-kter** [comment]
TotolaCommander -> show hidden files -> is there a **.stversion** folder in the synced folder?

**Comment by wweich** [comment]
How much data (size + number of files) are you syncing with how many devices?

Syncthing Android stores its index on the internal storage (inside the app directory). Inside the index is (of cause) all the information of the local files but also the information which was last send from each of the remote devices (this is new) and the some temp data. While syncthing is running there is also the temp index for different sorting which is needed for the "new" delta indexes.
This can all add up to 300MB when syncing much data/many files especially when syncthing doesn't come around to compact the DB.

**Comment by capi** [comment]
I also think this is about the internal database of Syncthing, as outlined by @wweich. We don't do any special logging on Android to the internal storage, so it has to be something related to the native part. Also, 300MB might actually be not that much, depending on number of folders, files, and peers.

**Comment by randoogle** [comment]
Dang, Gmail hid your replies from me under some forums folder. No there
isn't an .stversions folder. I've since disconnected the one folder, since
I can't sacrifice that much internal storage. It's stored on my sd card, so
an stversions folder shouldn't have made much difference. Here's what I
think is the problem: the share I'm connecting to has over 200GB of data,
so I used an stignore to narrow it down to only one sub folder. I think
that the app data index is related to the total size of the share and not
simply the subset of data you're syncing.
